static/src/js/topic_list.js

```diff
@@ -1,5 +1,5 @@
 /***
- * Topic List Design 1
+ * Topic List Design 2
```

Comment:
There is no more design 1 and 2 in this commit. Call it Topic List and remove the reference to design 1